rajawali/src/main/java/org/rajawali3d/materials/shaders/AShaderBase.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013 Dennis Ippel
```

Comment:
was going to add  comment and then changed my mind, the copyright line got changed by mistake, commited a change to fix the typo.